### ISSUE TITLE
Help on moco server stuck

### ISSUE THREAD

**Issue by wxjlibra** [dreamhead/moco#147]
I was making a gradle task to do a whole bunch of stuff about automatically running integration test.
My process is within a single customised gradle task. 
First start moco server with typical command with
'java -jar moco-runner-0.10.2-standalone.jar http -p 12306 -c test_data/b2/versions/b2_moco.json'
After that I'm running a cucumber test(integration test against it) 
However the moco server became not responding after sucessfully returning several response. And after that all my integration is telling me 504 error.
Meanwhile. If I run moco server separately and in another task run Integration it works well.
Do you know what might be the root cause for this issue. Thanks!

**Comment by dreamhead** [comment]
How do you start Moco server? in Gradle task?

**Comment by wxjlibra** [comment]
yes. a single task first to start moco server than to run it against it

**Comment by dreamhead** [comment]
Show me your Gradle configuration please.
